doc/getting_started.md

```diff
@@ -0,0 +1,85 @@
+# Getting started
+
+This document will help guide you through the setup of PyDotBot connected to
+a DotBot gateway and a DotBot robot.
+
+## Prerequisites
+
+Make sure you have access to an nRF DK board (nrf52833dk, nrf52840dk or
+nrf5340dk) and to a DotBot (v1 or v2).
+
+Follow the instructions in the
+[DotBot firmware getting started page][dotbot-firmware-getting-started].
+
+## Install PyDotBot
+
+Use pip to install the latest version of PyDotBot from [pypi][pydotbot-pypi]:
```

Comment:
maybe you need to specify valid python versions? I don't know.

Reply:
The supported Python versions are documented in the package itself normally